appveyor.yml

```diff
@@ -86,3 +82,11 @@
     name: NuGet public feed
     on:
       branch: master
+  - provider: GitHub    
+    release: $(GitVersion_NuGetVersion)
+    description: 'Release v$(GitVersion_NuGetVersion)'
+    auth_token:
+      secure: 2ijF4OfPYfIUTunubL/Z04ogADL9sx/gKW1jMRNJnAa1OpHZpSvCKmP0gvNs8C7h
```

Comment:
and this of course is encrypted with a private key exchanged only with appveyor, right?